src/util.js

```diff
@@ -2,26 +2,37 @@ import fs from 'fs';
 import { readFile } from 'fs/promises';
 import chalk from 'chalk';
 import * as dotenv from 'dotenv';
+import * as db from './database.js';
 
 dotenv.config();
 
+const ACTOR_SETTING_NAMES = ['username', 'avatar', 'displayName', 'description'];
+const IS_ACCOUNT_FILE_IMPORTED = 'isAccountFileImported';
```

Comment:
would it make sense for "devex" to move stuff related to upgrades/migrations into a separate util-like file? Like, we basically never want to get rid of code that relates to making sure that upgrades are smooth, but on the other hand we also want to minimize any given dev's exposure to this kind of code because _theoretically_ it should be set-and-forget.

I'm aware that in general util.js as a junk drawer is a problem brewing, but this feels like a clean delineation for "a kind of thing to put in a separate place"

Reply:
I've added a new file since this comment was made called `boot.js` which, as the name implies, is a bunch of procedures that should run once when the application boots. It's only used for migrating data around and I don't anticipate it having much use beyond that, but it does mean that `util.js` can be dedicated to utilities (which is still kind of a junk drawer, but a less crowded one).